code/go/0chain.net/chaincore/chain/protocol_block.go

```diff
@@ -397,8 +397,8 @@ func (c *Chain) finalizeBlock(ctx context.Context, fb *block.Block, bsh BlockSta
 					zap.Error(err),
 					zap.Int64("round", fb.Round),
 					zap.String("hash", fb.Hash))
-				EventsComputationTimer.Update(time.Since(ts).Microseconds())
-				return err //do not remove events in case of error
+				// EventsComputationTimer.Update(time.Since(ts).Microseconds())
+				// return err //do not remove events in case of error
```

Comment:
we should return here, otherwise we will remove the events from the block and we will skip this events

Reply:
Oh, it's a new block event, not all events. So should keep it, but avoid adding new block event.